package.json

```diff
@@ -38,7 +38,7 @@
     "date-fns": "2.16.1",
     "ethereumjs-abi": "0.6.8",
     "ethereumjs-util": "6.0.0",
-    "ethers": "4.0.48",
+    "ethers": "5.0.26",
```

Comment:
Maybe explicitly install `@ethersproject/*` stuff rather than `ethers`.

Reply:
Yeah, just don't know which ones we need at the moment 😄

Reply:
As of now:
* `@ethersproject/bignumber`
* `@ethersproject/address`
* `@ethersproject/providers`
* `@ethersproject/transactions`
* `@ethersproject/bytes`
* `@ethersproject/abstract-provider`
* `@ethersproject/units`
* `@ethersproject/wallet`